keyboards/yunzii/common/smart_ble.h

```diff
@@ -0,0 +1,22 @@
+// /* Copyright 2024 Jacky
+//  *
+//  * This program is free software: you can redistribute it and/or modify
+//  * it under the terms of the GNU General Public License as published by
+//  * the Free Software Foundation, either version 2 of the License, or
+//  * (at your option) any later version.
+//  *
+//  * This program is distributed in the hope that it will be useful,
+//  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  * GNU General Public License for more details.
+//  *
+//  * You should have received a copy of the GNU General Public License
+//  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//  */
+
+#pragma once
+#include "quantum.h"
+
+void sc_ble_battary(uint8_t batt_level);
+void WIRELESS_START(uint32_t mode);
+void WIRELESS_STOP(void);
```

Comment:
Nitpick.

These should probably be named `wireless_start` and `wireless_stop`.

Reply:
thanks.